srcpkgs/syncthing-gtk/template

```diff
@@ -1,19 +1,29 @@
 # Template file for 'syncthing-gtk'
 pkgname=syncthing-gtk
 reverts="0.14.36_1"
-version=0.9.4.4+ds+git20220108+9023143f8b93
-revision=2
+reverts="0.9.4.4+ds+git20220108+9023143f8b93_1"
+reverts="0.9.4.4+ds+git20220108+9023143f8b93_2"
```

Comment:
All versions we want to revert need to be in one `reverts` variable, otherwise only the last one would be used.